docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -80,28 +80,41 @@ For example, the application processes only 4 video streams, so there is no need
 
 @endsphinxdirective
 
-For the *explicit* usage, you can limit the batch size using  "BATCH:GPU(4)",  where 4 is the number of requests running in parallel.
+For the *explicit* usage, you can limit the batch size using "BATCH:GPU(4)",  where 4 is the number of requests running in parallel. Notice that since the CPU doesn't support the `ov::optimal_batch_size` yet, it works with the auto-batching only when explicit batch size is specified, for example "BATCH:CPU(16)". 
```

Comment:
do we need to write specifically about CPU? Guide must be generic.

`Notice if device doesn't support the `ov::optimal_batch_size` yet, it works with the auto-batching only when explicit batch size is specified, for example "BATCH:<device>(16)"`